tests/ord-service/tests/subscription_flow_test.go

```diff
@@ -961,9 +961,12 @@ func TestConsumerProviderFlow(stdT *testing.T) {
 		// After successful subscription from above we call the director component with "double authentication(token + user_context header)" in order to test claims validation is successful
 		consumerToken := token.GetUserToken(stdT, ctx, conf.ConsumerTokenURL+conf.TokenPath, conf.ProviderClientID, conf.ProviderClientSecret, conf.BasicUsername, conf.BasicPassword, claims.SubscriptionClaimKey)
 		consumerClaims := token.FlattenTokenClaims(stdT, consumerToken)
-		consumerClaimsWithEncodedValue, err := sjson.Set(consumerClaims, "encodedValue", "test+n%C3%B8n+as%C3%A7ii+ch%C3%A5%C2%AEacte%C2%AE")
-		require.NoError(t, err)
-		headers := map[string][]string{subscription.UserContextHeader: {consumerClaimsWithEncodedValue}}
+		clientID := gjson.Get(consumerClaims, "client_id")
+		subaccountID := gjson.Get(consumerClaims, "subaccountid")
+		userName := gjson.Get(consumerClaims, "user_name")
+		subdomain := gjson.Get(consumerClaims, "subdomain")
```

Comment:
You can load the properties from the values.yaml and use them in the tests because in local and productive setup they are different